qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -416,21 +425,27 @@ def add_dockwidget(self, plugin):
 
     # ----------------------- Layout ---------------------------------
 
-    def setup_layout(self):
-        """Assume this is a first run of the application and set layouts
-        accordingly"""
-        self.setup_default_layouts()
+    def save_layout_settings(self):
+        """Save the current layout to configuration"""
+        CONF.set("Layout/Layout", self.saveState())
+
+    def load_layout_settings(self):
+        """Load a layout for the child widgets from the user config, if no user default exists load the default"""
```

Comment:
This doc-string is a bit long, can we split it into two lines?

Reply:
```suggestion
        """Load a layout for the child widgets from the user config, 
        if no user default exists load the default"""
```